src/kOS/Suffixed/KUniverseValue.cs

```diff
@@ -325,13 +389,13 @@ private void LaunchShip(CraftTemplate ship, string launchSiteName)
                 {
                     SafeHouse.Logger.LogError("Could not launch vessel, did not pass preflight...");
                 });
-            if (launchSiteName.Equals("runway", System.StringComparison.OrdinalIgnoreCase))
+            if (launchSiteName.ToString().Equals("runway", System.StringComparison.OrdinalIgnoreCase))
```

Comment:
I don't get what difference this edit makes.  Isn't ``launchSiteName`` already a string parameter passed in here?